<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export data</string>
  </property>
  <property name="windowIcon">
   <iconset theme="document-save">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>Output file</string>
     </property>
     <property name="buddy">
      <cstring>pathButton</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="pathEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Select output file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pathButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>../MultimapAWZ</normaloff>../MultimapAWZ</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="userLabel">
     <property name="text">
      <string>User</string>
     </property>
     <property name="buddy">
      <cstring>userCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="userCombo">
     <item>
      <property name="text">
       <string>All users</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status</string>
     </property>
     <property name="buddy">
      <cstring>statusCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="statusCombo">
     <item>
      <property name="text">
       <string comment="status">Any</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Done</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Undone</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="validationLabel">
     <property name="text">
      <string>Validation</string>
     </property>
     <property name="buddy">
      <cstring>validationCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="validationCombo">
     <item>
      <property name="text">
       <string comment="validation">Any</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Valid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Invalid</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lockedLabel">
     <property name="text">
      <string>Lock</string>
     </property>
     <property name="buddy">
      <cstring>lockedCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="lockedCombo">
     <item>
      <property name="text">
       <string extracomment="locked">Any</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Locked</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unlocked</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="fromCheck">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDateTimeEdit" name="fromDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="toCheck">
     <property name="text">
      <string>To</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDateTimeEdit" name="toDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>96</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pathEdit</tabstop>
  <tabstop>pathButton</tabstop>
  <tabstop>userCombo</tabstop>
  <tabstop>statusCombo</tabstop>
  <tabstop>validationCombo</tabstop>
  <tabstop>lockedCombo</tabstop>
  <tabstop>fromCheck</tabstop>
  <tabstop>fromDate</tabstop>
  <tabstop>toCheck</tabstop>
  <tabstop>toDate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>fromDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>toDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
